src/asm2wasm.h

```diff
@@ -508,6 +508,7 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
       import->name = Name(std::string(import->name.str) + "$asm2wasm$import");
       import->kind = Import::Global;
       import->globalType = type;
+      import->globalMutable = false;
```

Comment:
false is the default anyhow, so no need to set this field in this file? (otherwise for consistency it should be set in wasm-s-parser too, but no need for either)